dpc/src/transaction/transition.rs

```diff
@@ -554,13 +574,13 @@ mod tests {
             let transaction = Testnet1::genesis_block().to_coinbase_transaction().unwrap();
             let transition = transaction.transitions().first().unwrap().clone();
             let transition_bytes = transition.to_bytes_le().unwrap();
-            assert_eq!(1389, transition_bytes.len());
+            assert_eq!(501, transition_bytes.len());
```

Comment:
Transitions no longer contain input/output proofs, and so the transition is smaller.